sycl/include/sycl/kernel_bundle.hpp

```diff
@@ -548,6 +574,24 @@ kernel_bundle<State> get_kernel_bundle(const context &Ctx,
   return get_kernel_bundle<State>(Ctx, Devs, {get_kernel_id<KernelName>()});
 }
 
+// For free functions.
+namespace ext::oneapi::experimental {
+template <auto *Func, bundle_state State>
+std::enable_if_t<ext::oneapi::experimental::is_kernel_v<Func>,
```

Comment:
```suggestion
std::enable_if_t<is_kernel_v<Func>,
```

You can simplify these because you are already in that namespace.  Same comment below.

Reply:
I've removed unneeded qualifiers.